enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/GitEnricher.java

```diff
@@ -61,6 +61,11 @@ public GitEnricher(MavenEnricherContext buildContext) {
                         if (id != null) {
                             annotations.put(Fabric8Annotations.GIT_COMMIT.value(), id);
                         }
+
+                        String gitOriginUrl = repository.getConfig().getString("remote", "origin", "url");
```

Comment:
should we offer the option to allow the user to set this as a parameter?

Reply:
Check now :)